src/components/Detail/Detail.js

```diff
@@ -1,73 +1,76 @@
 import React, {useEffect, useState} from 'react';
 import './Detail.scss'
 import ImageGallery from 'react-image-gallery';
-
-import * as ApiService from '../../ApiService/ApiService'
-import Bar from '../BottomBar/BottomBar';
+import * as ApiService from 'ApiService/ApiService'
+import Bar from 'components/BottomBar/BottomBar';
 
 export default function Detail(props) {
 
-  const [product,
-    setProduct] = useState({});
-  
-    let productId = props.match.params.id;
+    const [product,
+        setProduct] = useState({});
+
+    const productId = props.match.params.id;
 
     useEffect(() => {
         ApiService
             .get("/api/products", `/${productId}`)
             .then(res => {
                 setProduct(res);
             })
-    }, []); 
- 
+    }, []);
+
+    const imageUrl = `${process.env.REACT_APP_IMAGE}${product.id}`;
     const images = [
-      {
-        original: `http://localhost:8080/api/product_images/${productId}`,
-        thumbnail: `http://localhost:8080/api/product_images/${productId}`
-      },
-      {
-        original: `http://localhost:8080/api/product_images/${productId}`,
-        thumbnail: `http://localhost:8080/api/product_images/${productId}`,
-      },
-      {
-        original: `http://localhost:8080/api/product_images/${productId}`,
-        thumbnail: `http://localhost:8080/api/product_images/${productId}`,
-      }
+        {
+            original: `${imageUrl}`,
+            thumbnail: `${imageUrl}`
```

Comment:
How do you know you have exactly 3 images with same url/thumbnail? Why do you need this?